src/host/path_join.c

```diff
@@ -1,62 +1,56 @@
 /**
- * \file   path_join.c
- * \brief  Join two or more pieces of a file system path.
- * \author Copyright (c) 2002-2013 Jason Perkins and the Premake project
- */
+* \file   path_join.c
+* \brief  Join two or more pieces of a file system path.
+* \author Copyright (c) 2002-2013 Jason Perkins and the Premake project
+*/
 
 #include "premake.h"
 #include <string.h>
 
+#define JOIN_RELATIVE 0
+#define JOIN_ABSOLUTE 1
+#define JOIN_MAYBE_ABSOLUTE 2
```

Comment:
These are also defined in `path_isabsolute.c`, perhaps they should exist in a shared header to avoid issues later?